client/components/profile/Profile.jsx

```diff
@@ -55,6 +59,33 @@ Profile = class Profile extends Component {
           {showTeamPreview ? <ProfileTeamPreview /> : null }
 
           <PasswordEditor />
+          <div>
+            <Message
+              info size='large'
+              header='Help keep us afloat!'
+              content={`One of our goals is to keep the WWU Great
+                        Puzzle Hunt FREE so that EVERYONE can participate.
+                        To do that, we depend on donations from those who
+                        can afford them. If you are able, please consider
+                         making a small donation (suggested $5 students,
+                         $10 nonstudents) and/or buying a shirt (our only fund
+                         raiser)`}
+            />
+            </div>
+            <br/>
+            <div>
+            <LinkButton as='a'
+              href={link}
+              size='large'  content='Donate Online'
+              icon={<Icon name='heart'/>}
+              color="green"
+            />
+            <LinkButton as='a' href="/gear"
+              size="large" color="orange" target="_blank"
```

Comment:
May want to drop the `target="_blank"` since this is just directing to another page on the same site

Reply:
I thought about that, and decided to leave the target in there. This matches what we have elsewhere, and I think it's a good idea not to interrupt the flow that a user has with registration/verification/team joining or whatever.

Reply:
great point. i agree with your assessment